soul-common/src/main/java/org/dromara/soul/common/enums/PluginEnum.java

```diff
@@ -148,4 +149,13 @@ public static PluginEnum getPluginEnumByName(final String name) {
                 .filter(pluginEnum -> pluginEnum.getName().equals(name))
                 .findFirst().orElse(PluginEnum.GLOBAL);
     }
+
+    /**
+     * get plugin names.
+     *
+     * @return List string
+     */
+    public static List<String> getPluginNames() {
```

Comment:
Can we change the name? It seems too generic. : )

Reply:
ok